examples/specs/line_impute_transform_frame.vl.json

```diff
@@ -8,18 +8,34 @@
       {"x": 1, "y": 55, "c": 1},
       {"x": 2, "y": 81, "c": 0},
       {"x": 2, "y": 53, "c": 1},
-      {"x": 3, "y": 19, "c": 0}]},
+      {"x": 3, "y": 19, "c": 0}
+    ]
+  },
   "transform": [
-    {"impute": "y",
-     "key": "x",
-     "groupby": ["c"],
-     "method": "mean",
-     "frame": [-2, 2]
-    }],
+    {
+      "impute": "y",
+      "key": "x",
+      "groupby": ["c"],
+      "method": "mean",
+      "frame": [-2, 2]
+    }
+  ],
   "mark": "line",
   "encoding": {
-    "x": {"field": "x", "type": "quantitative", "scale": {"nice": 1}},
-    "y": {"field": "y", "type": "quantitative"},
-    "color": {"field": "c", "type": "nominal"}
+    "x": {
```

Comment:
The old encoding here was already good and concise.  I just don't like that you don't do new lines for closing brackets.